plugins/cilium-docker/main.go

```diff
@@ -50,6 +51,16 @@ connected to a Docker network of type "cilium".`,
   docker run --net my_network hello-world
 `,
 	Run: func(cmd *cobra.Command, args []string) {
+		if ciliumAPI == "" {
+			// Check if environment variable points to socket
+			e := os.Getenv(defaults.SockPathEnv)
+			if e == "" {
+				// If unset, fall back to default value
+				e = defaults.SockPath
+			}
+			ciliumAPI = "unix://" + e
```

Comment:
Code is correct. Can you move the above into a new function `DefaultSockPath() string` in `pkg/client/client.go`? So we can share the same code for this and `client.NewClient()` as well. The change itself is absolutely correct.